api/base/middleware.py

```diff
@@ -19,6 +21,7 @@ def process_exception(self, request, exception):
         """If an exception occurs, rollback the current transaction
         if it exists.
         """
+        sentry_exception_handler(request=request)
```

Comment:
Raven does its own checks to ensure that the client is active. captureException (used internally) returns None is no dsn key is provided, and when the API app is loaded, the message `Raven is not configured (logging is disabled). Please see the documentation for more information.` appears on the screen.